front/item_device.common.form.php

```diff
@@ -54,10 +54,21 @@
 if (isset($_POST["id"])) {
    $_GET["id"] = $_POST["id"];
 } else if (!isset($_GET["id"])) {
-   $_GET["id"] = -1;
+   $_GET["id"] = "";
 }
 
-if (isset($_POST["purge"])) {
+if (isset($_POST["add"])) {
+   $item_device->check(-1, CREATE, $_POST);
+   if ($newID = $item_device->add($_POST)) {
+      Event::log($newID, get_class($item_device), 4, "setup",
+                 sprintf(__('%1$s adds the item %2$s'), $_SESSION["glpiname"], $_POST["name"]));
```

Comment:
A device item does not have a name.
```suggestion
                 sprintf(__('%1$s adds an item'), $_SESSION["glpiname"]));
```

```
  *** PHP Notice(8): Undefined index: name
  Backtrace :
  front/item_device.common.form.php:64               
  front/item_devicesimcard.form.php:36               include()
```